### PR TITLE
Jack Sneed pull request for Task6

### PR DESCRIPTION
**Pull request by JackSSS**

### CODE REVIEW

File: students/JackSneed/session03/list_lab.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+
+def fruit_list():
+
+    #List Lab part 1
```
Comment (am0k) [inline students/JackSneed/session03/list_lab.py]: Remember to add an empty space after you start a comment.

File: students/JackSneed/session03/list_lab.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    fruit = [u"Apples", u"Pears", u"Oranges", u"Peaches"]
+    print(fruit)
+
+    new_fruit = raw_input(u"Name a new fruit: ")
+    fruit.append(new_fruit)
+    print(fruit)
+
+    user_pick = int(raw_input(u"Pick a number 1-5: "))
+    fruit_chosen = fruit[user_pick - 1]
+    print(u"The number %i fruit on the index is %s." % (user_pick, fruit_chosen))
```
Comment (am0k) [inline students/JackSneed/session03/list_lab.py]: If you have a working Python Flake8 Lint working in Sublime, it should tell you that lines of code should not be longer than 79 characters.

File: students/JackSneed/session03/list_lab.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    user_pick2 = (raw_input(u"Choose a fruit from the index to remove: "))
+    fruit.remove(user_pick2)
+    print(fruit)
+
+    #List Lab part 3
+
+    for item in fruit[:]:
+        answer = u""
+        while answer != u"yes" and answer != u"no":
+            answer = raw_input(u"Do you like %s? (Enter yes or no): " % (item.lower()))
```
Comment (am0k) [inline students/JackSneed/session03/list_lab.py]: This line too long as well.

File: students/JackSneed/session03/list_lab.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+    #List Lab part 4
+
+    newest_fruit = fruit[:]
+    newest_fruit = [item[::-1] for item in fruit]
+    del fruit[-1]
+    print(fruit)
+    print(newest_fruit)
+
+
```
Comment (am0k) [inline students/JackSneed/session03/list_lab.py]: Flake8 linters should tell you that you have one too many empty lines in your code here.

**Comment by am0k** [comment]
Great work, all you need to improve are minor style errors, and maybe improving the user experience when it comes to running your script from terminal. It's hard to follow it without having your code open in Sublime.

**Comment by JackSSS** [comment]
Thank Ulises!
